.prettierrc.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
   ...require('@spotify/prettier-config'),
+  arrowParens: 'always',
+  printWidth: 100,
+  singleQuote: true,
+  trailingComma: 'all',
```

Comment:
These are already covered by parent `@spotify/prettier-config`

https://github.com/spotify/web-scripts/blob/master/packages/prettier-config/index.js

This brings no value.